src/resource-managers/expression-manager.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Expression Manager module.
+ * @module resource-managers/expression-manager
+ * @see module:resource-managers/resource-manager
+ */
+'use strict';
+
+var ResourceManager = require('./resource-manager');
+
+/**
+ * Expression Manager Object
+ *
+ * Class representing an Expression manager.
+ * Expressions are formulas that execute on the SolveBio API. They return a
+ * single value or list of values with a specific data type.
+ *
+ * @constructor
+ * @augments ResourceManager
+ * @requires module:resource-managers/resource-manager
+ */
+
+var ExpressionManager = function(solveBio, expression, dataType, isList) {
```

Comment:
Do we need the `dataType` or `isList` on instantiation?
I would have put `expression` and `data` which are parameters of the expression.
`dataType` or `isList` are more evaluation parameters.
Then we would use the following syntax:
```js
var expression = SolveBio.Expression(my_expression, my_data);
expression.evaluate(dataType, isList);
```